apps/studio.giselles.ai/migrations/0027_funny_infant_terrible.sql

```diff
@@ -0,0 +1,23 @@
+DROP TABLE "builds";--> statement-breakpoint
```

Comment:
@toyamarinyon The migration encountered an error due to an FK constraint. Could you modify the query to delete the FK first?

Here is an example:

```sql
ALTER TABLE edges DROP CONSTRAINT edges_build_db_id_builds_db_id_fk;
ALTER TABLE nodes DROP CONSTRAINT nodes_build_db_id_builds_db_id_fk;
ALTER TABLE requests DROP CONSTRAINT requests_build_db_id_builds_db_id_fk;
ALTER TABLE trigger_nodes DROP CONSTRAINT trigger_nodes_build_db_id_builds_db_id_fk;

DROP TABLE builds;
```